internal/tasks/fixtures/account_management_error.yaml

```diff
@@ -0,0 +1,2 @@
+accounts:
+- name: ""
```

Comment:
I don't see this file being used in the test. Probably a leftover before this test got moved to `internal/drivers/basic`?